api/hasura/auth.ts

```diff
@@ -45,6 +45,11 @@ export default async function handler(req: VercelRequest, res: VercelResponse) {
       return;
     }
 
+    if (!req.headers?.authorization) {
+      return res.status(200).json({
+        'X-Hasura-Role': 'anon-user',
```

Comment:
Maybe setup some per-ip rate limiting for anon-user role?

https://hasura.io/learn/graphql/hasura-advanced/security/4-rate-limit/

Reply:
YAH THAT WOULD BE SICK